core/webengine.py

```diff
@@ -259,6 +269,12 @@ def eventFilter(self, obj, event):
 
         return super(QWebEngineView, self).eventFilter(obj, event)
 
+    def linkHovered(self, url):
```

Comment:
Python function should use underline style, like ```link_hovered```, camel style only use if library use camel style, such as PyQt